docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -609,106 +612,347 @@ simulants are dying at the expected rates.
+chose to model the following relationships:
+
+* Households/people move in each of the above types depending on their demographics (age, sex, race/ethnicity).
+* People who move into GQ, move into a GQ category (institutional or non) that depends on their demographics.
+* People who join an existing household, join with a relationship that depends on their demographics.
+* People who move, are more likely to move to certain PUMAs (primarily close by) depending on the MIGPUMA they currently live in.
+
+All other correlations do not exist.
+For example:
+
+* Whether or not people move is unaffected by their current living arrangement.
+* The location people move to is unaffected by their demographics.
+* The location people move to is independent of the living arrangement they move into.
+* People who do not move to one of the most likely PUMAs according to their MIGPUMA
+  move to a PUMA totally independent of their current location.
+* And so on.
+
+Move rates by type
+''''''''''''''''''
+
+We calculate the rate of household moves per household-year, stratified by the demographics
+of the reference person.
+
+Likewise, we calculate the rate of each individual move type (GQ person, new household, non-reference person)
+per person-year, stratified by demographics.
+
+Relationship
+''''''''''''
+
+In household moves, relationships are unchanged.
+In new-household moves, the relationship in the new household is always "reference person."
+Therefore, there are two move types that require a choice of relationship attribute:
+GQ moves (where the relationship attribute represents institutional vs non-institutional),
+and non-reference person moves.
+
+For each of these move types, we calculate **the proportion of movers of that type** who
+have each relationship, stratified by individual demographics -- age group, sex, and race/ethnicity.
+
+To address sample size issues, which are present especially for the less-common relationships in the smallest
+race/ethnicity groups, we perform a smoothing procedure (described below) in each demographic group
+(e.g. 0-15 year old females of NHOPI race/ethnicity)
+*twice*: first using information from the distribution in the same group without stratification by race/ethnicity
+(e.g. all 0-15 year old females),
+and then using information from the distribution in the group stratified only by age
+(e.g. all 0-15 year olds).
+We refer to these groups, which are less stratified and have larger sample size, as the "reference group"
+in the smoothing procedure description below.
+We do not include a third smoothing pass with the fully unconditional distribution (all people regardless of age),
+because there are logical relationships between age and relationship -- for example,
+children should never be spouses.
+
+Our smoothing procedure is as follows:
+
+* Take the total probability of all demographic + relationship combinations which have less than **30** sample size,
```

Comment:
I'd be interested to ask you over zoom if i'm understanding this correctly -- but not necessary for implementation